src/main/resources/application.yaml

```diff
@@ -153,3 +155,15 @@ hapi:
 #  protocol: 'http'
 #  schema_management_strategy: CREATE
 #  username: SomeUsername
+#security:
+#  ignored: none
+keycloak:
+  auth-server-url: http://localhost:8180/auth/
+  realm: fhir-core
+  resource: fhir-core-server
+  credentials:
+    secret: b7747a19-f72f-4906-8892-8438ce2492be
```

Comment:
These too should be moved to secrets file. @rehammuzzamil Are these credentials for a local keycloak instance?

Reply:
@ekigamba  yes these creds are for local keycloak instance.

Reply:
Cool @rehammuzzamil , nice work

Reply:
@rehammuzzamil can we load this from a properties file or something. 

@bennsimon @manutarus does defining these in a separate file help in encrypting them?

Reply:
@dubdabasoduba  these keycloak configurations need to be defined in a properties file either `application.yml` or `application.properties`. In our case, we have an` _application.yml file_`